app/config/security.yml

```diff
@@ -14,19 +13,16 @@ security:
             - ROLE_USER
 
         ROLE_SUPER_ADMIN:
-            - ROLE_SONATA_ADMIN
-            - ROLE_ALLOWED_TO_SWITCH
+            - ROLE_ADMIN
 
+    # todo use with bcrypt
     encoders:
-        Sylius\Component\User\Model\UserInterface: sha512
+        Sil\Bundle\UserBundle\Entity\User: plaintext
```

Comment:
"plaintext"  :gun:

Reply:
It is easier for test to have plaintext, bcrypt will be enable in another user story (6327)